pp3/module/Application/src/Application/Controller/PluginController.php

```diff
@@ -349,12 +350,13 @@ private function _sendApprovalEmail($plugin) {
         $mail->setSubject('NetBeans plugin waiting for approval: '.$plugin->getName());
         $transport = new Mail\Transport\Sendmail();
         $i=0;
-        foreach($this->_verifierRepository->findAll() as $verifier) {
+        foreach($this->_userRepository->findVerifier() as $verifier) {
             if ($i == 0) {
-                $mail->addTo($verifier->getUserId());
+                // @todo: get an explanation why the first verifier is special cased
+                $mail->addTo($verifier->getEmail());
                 $i++;
             } else {
-                $mail->addBcc($verifier->getUserId());
+                $mail->addBcc($verifier->getEmail());
             }
```

Comment:
@jpirek Why don't we simply add all verifiers on BCC: list?

Reply:
Yes, that makes sence, it will speed things up.